zerver/lib/events.py

```diff
@@ -1136,52 +1117,58 @@ def do_events_register(
     # Fill up the UserMessage rows if a soft-deactivated user has returned
     reactivate_user_if_soft_deactivated(user_profile)
```

Comment:
(I’m assuming this and everything above doesn’t need to be repeated.)

Reply:
Correct.